source/opt/instruction.h

```diff
@@ -328,6 +342,10 @@ class Instruction : public utils::IntrusiveNodeBase<Instruction> {
   bool IsReadOnlyVariableShaders() const;
   bool IsReadOnlyVariableKernel() const;
 
+  // Returns true if it is valid to use the result if |inst| as the base
+  // pointer for a load or store.
```

Comment:
Should say that this will return true if using physical addressing or if relaxed-logical addressing rules are satisfied.

Reply:
done.